tests/src/Common/test_runtime/project.json

```diff
@@ -2,9 +2,6 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "2.0.0-beta-25129-02",
     "Microsoft.DotNet.CoreCLR.TestDependencies": "1.0.0-prerelease",
-    "jit-dasm": "0.0.1.4",
-    "cijobs": "0.0.1.2",
-    "jit-analyze": "0.0.1.1"
```

Comment:
What problem are these lines causing?  I don't think we can just remove these.  There may be a better way to accomplish what they are doing (making certain CI jobs work) but we should understand why they don't work on Unix builds before looking at that.

@sbomer @russellhadley